sentinel-ai-models-simple-openai/src/test/java/com/phonepe/sentinelai/models/SimpleOpenAIModelStreamingTest.java

```diff
@@ -172,6 +146,82 @@ void testAgent(final WireMockRuntimeInfo wiremock) {
         log.info("Session stats: {}", stats);
     }
 
+    @NotNull
+    private static TestAgent setupAgent(final WireMockRuntimeInfo wiremock, final JsonMapper objectMapper, final OkHttpClient httpClient, final ExecutorService executor) {
```

Comment:
Formatting

Reply:
Also, move private functions below public ones. Please be mindful of making the same mistakes on multiple PRs